test/e2e/scenario/recorder.scenario.ts

```diff
@@ -63,7 +62,7 @@ describe('recorder', () => {
       )
     })
 
-  describe('snapshot', () => {
+  describe('full snapshot', () => {
```

Comment:
is full snapshot equivalent to document serialization?
if yes, what about renaming these references as well?

Reply:
A full snapshot is a FullSnapshot record, which contain the serialized document and the document scroll offset. I think we should keep the same naming to keep consistency between player and recorder.

Reply:
I think we should not hesitate to tweak the naming on player and recorder side if it makes more sense to us.
You have a better grasp on that so I'll let you decide what is making the more sense.

Reply:
I would agree that we could improve the protocol namings, but I would not do this incrementally by renaming only FullSnapshot and keeping IncrementalSnapshot for example. I think it would be worth to discuss more about this with an RFC when we feel the need for it.